src/Tribe/Tickets.php

```diff
@@ -2494,6 +2503,12 @@ public function send_tickets_email_for_attendee( $to, $tickets, $args = [] ) {
 				// Mark attendee ticket email as being sent for each attendee ticket.
 				foreach ( $tickets as $attendee ) {
 					$this->update_ticket_sent_counter( $attendee['attendee_id'], $this->attendee_ticket_sent );
+
+					$this->update_attendee_activity_log( $attendee['attendee_id'], [
```

Comment:
* :no_entry_sign: Opening parenthesis of a multi-line function call must be the last content on the line
* :no_entry_sign: Only one argument is allowed per line in a multi-line function call

<sub>hash: 0c0f8b1cb79823eed128da0779224f05</sub>